mangaki/mangaki/views.py

```diff
@@ -919,8 +974,8 @@ def update_evidence(request):
             evidence.needs_help = needs_help
             evidence.save()
 
-    if suggestion_id:
-        return redirect('fix-suggestion', suggestion_id)
+    if 'next' in request.GET:
+        return redirect(request.GET['next'])
```

Comment:
We should be wary of [unvalidated redirections](https://www.owasp.org/index.php/Unvalidated_Redirects_and_Forwards_Cheat_Sheet), e.g. `?next=https://my.evil.website.com/`.